core/encoding.go

```diff
@@ -0,0 +1,170 @@
+// Copyright 2022 Democratized Data Foundation
```

Comment:
```suggestion
// Copyright 2023 Democratized Data Foundation
```

Reply:
My bad, I didnt realize I missed that, thanks

- [ ] 2023